packages/next/src/server/lib/router-utils/block-cross-site.ts

```diff
@@ -23,6 +24,9 @@ export const blockCrossSite = (
       res.statusCode = 403
     }
     res.end('Unauthorized')
+    warnOnce(
+      `Blocked cross-origin request to /_next/*. To allow this, configure "allowedDevOrigins" in next.config`
+    )
```

Comment:
Is this supposed to include a "Read more" link to the page that you've added in this PR?